### ISSUE TITLE
Add spec to allow clntent signing

### ISSUE THREAD

**Issue by kelson42** [openzim/libzim#40]

**Comment by mgautierfr** [comment]
I'm suppose you are speaking about "content signing" no ?

**Comment by kelson42** [comment]
@mgautierfr Yes, this is not urgent, but at middle term I support we should have a solution to sign digitaly the ZIM files.

**Comment by kelson42** [comment]
@mofosyne I'm not sure what exactly would be allowed  to verified by building the catalog within the libzim!

**Comment by mofosyne** [comment]
@kelson42 it would specifically inform the user that if they trust the integrity of their libzim executable and their zim file is old enough and notable enough to be included into libzim then they can trust it.

It will however not let them know if a newer zim file is trustable, even if it's trusted by the community... until they update their libzim executable.

It's basically a hack, that works usually because we trust our package manager to not be compromised.

**Comment by kelson42** [comment]
> @kelson42 it would specifically inform the user that if they trust the integrity of their libzim executable and their zim file is old enough and notable enough to have it's hash be included into libzim then they can trust it.

if they download from library.kiwix.org, the authenticity is anyway secure via https, so what you propose seems to me a ver imperfect hack not adding much value
